ui/packages/app/src/main.ts

```diff
@@ -192,6 +192,7 @@ function create_custom_element() {
 		root: ShadowRoot;
 		wrapper: HTMLDivElement;
 		_id: number;
+		theme: string;
```

Comment:
All the changes here are so that the `Plot` component has access to the current theme